### PR TITLE
fix(FEC-10144): player causes form submit on button click

### PR DESCRIPTION
**Pull request by shalommeoded**
### Description of the Changes

When the player is nested within a form when clicking any button that is in the control bar we get redirected to any form referral.

### CheckLists

- [X] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [X] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/button/button.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const COMPONENT_NAME = 'Button';
+
+/**
+ * Button component
+ *
+ * @class Button
+ * @example <Button/>
+ * @extends {Component}
+ */
+class Button extends Component {
```
Comment (OrenMe) [inline src/components/button/button.js:14]: as we don't need any lifecycle management we can use function component

**Comment by OrenMe** [comment]
@shalommeoded what do you mean by collide with the `withAnimation` HoC?

File: src/components/forward/forward.js [hunk 2]
```diff
@@ -66,14 +67,15 @@ class Forward extends Component {
     return (
       <div className={[style.controlButtonContainer, style.noIdleControl].join(' ')}>
         <Tooltip label={this.props.forwardText}>
-          <button
+          <Button
+            type={'button'}
```
Comment (yairans) [inline src/components/forward/forward.js:71]: is needed? 
already got it in Button no?

File: src/components/rewind/rewind.js [hunk 3]
```diff
@@ -65,14 +66,15 @@ class Rewind extends Component {
     return (
       <div className={[style.controlButtonContainer, style.noIdleControl].join(' ')}>
         <Tooltip label={this.props.rewindText}>
-          <button
+          <Button
+            type={'button'}
```
Comment (yairans) [inline src/components/rewind/rewind.js:70]: same Q in forward.js

File: src/components/button/button.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+//@flow
+import {h} from 'preact';
+import {forwardRef} from 'preact/compat';
+
+const COMPONENT_NAME = 'Button';
```
Comment (yairans) [inline src/components/button/button.js:5]: need to export it in `src/components/index.js`

Comment (RoyBregman) [reply src/components/button/button.js:5]: Not in src/components/index.js i think. in src/components/button/index.js

File: src/components/forward/forward.js (hunk 2 above)
Comment (RoyBregman) [reply src/components/forward/forward.js:71]: I don't think type button is needed for button tag

File: src/components/rewind/rewind.js (hunk 3 above)
Comment (shalommeoded) [reply src/components/rewind/rewind.js]: I missed that, they were from first PR, I forgot to remove them

File: src/components/button/button.js (hunk 4 above)
Comment (shalommeoded) [reply src/components/button/button.js:5]: Done

**Pull request merged**